BETA: Update jonathancarrillo.is-a.dev

bi0hazarDD opened the pull request:
Updated `jonathancarrillo.is-a.dev` using the [dashboard](https://manage.is-a.dev).

bi0hazarDD committed:
feat(domain): jcs.is-a.dev

WilliamDavidHarrison commented on domains/jonathancarrillo.json:
```diff
@@ -4,6 +4,6 @@
     },
     "record": {
-        "URL": "http://johny-portfolio.s3-website.eu-north-1.amazonaws.com"
+            "CNAME": "http://johny-portfolio.s3-website.eu-north-1.amazonaws.com/"
```
Please use a valid CNAME here. You need to remove the `http://` and `/`

WilliamDavidHarrison commented:
Also, please resolve any merge conflicts.

bi0hazarDD committed:
Update jonathancarrillo.json

was changed by accident because earlier I was trying to do it through aws Buckets , and that was an old PR, i'm hosting my static website on gh pages now

bi0hazarDD committed:
Merge pull request #1 from bi0hazarDD/bi0hazarDD-patch-1-1

Bi0hazar dd patch 1 1

bi0hazarDD committed:
Merge branch 'is-a-dev:main' into main

bi0hazarDD committed:
Create _github-pages-challenge-bi0hazarDD.jonathancarrillo.json

github verification

bi0hazarDD committed:
Create _github-pages-challenge-bi0hazarDD.jonathancarrillo.json

verification gh pages

bi0hazarDD committed:
Merge pull request #3 from bi0hazarDD/bi0hazarDD-patch-1-3

Bi0hazar dd patch 1 3

bi0hazarDD committed:
Merge pull request #2 from bi0hazarDD/bi0hazarDD-patch-1-2

Bi0hazar dd patch 1 2

bi0hazarDD committed:
Merge branch 'is-a-dev:main' into main

Commit:
Rename _github-pages-challenge-bi0hazarDD.jonathancarrillo.json to _github-pages-challenge-bi0hazardd.jonathancarrillo.json

Commit:
Merge branch 'main' into main
